SQL/updates/30-31.sql

```diff
@@ -0,0 +1,18 @@
+# Updates DB from 30 to 31 -AffectedArc07
+# Creates new tables in preparation for karma conversion
+
+# Rename old tables
+RENAME TABLE `karma` TO `karma_log`;
+RENAME TABLE `karmatotals` TO `karma_totals`;
+
+# Create new table to track new purchases
+CREATE TABLE `karma_purchases` (
+	`id` INT(11) NOT NULL AUTO_INCREMENT,
+	`ckey` VARCHAR(32) NOT NULL COLLATE 'utf8_general_ci',
+	`purchase` VARCHAR(64) NOT NULL COLLATE 'utf8_general_ci',
+	`purchase_time` DATETIME NOT NULL DEFAULT current_timestamp(),
+	PRIMARY KEY (`id`) USING BTREE,
```

Comment:
InnoDB only uses b-tree indexes. Specifying index type is redundant. Both here and in the already existing schema. 

Might as well leave this in so it looks prettier I guess?

Reply:
I would rather keep consistency, and this is what shows as the create code for the table.
![image](https://user-images.githubusercontent.com/25063394/156364954-0332f4ac-069d-47b7-876f-a2f94f589e6a.png)

*obviously the increment offset isnt included*